.github/workflows/app-samples-CI.yml

```diff
@@ -28,10 +28,10 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - name: Set up JDK 1.8
+    - name: Set up JDK 11
       uses: actions/setup-java@v1
       with:
-        java-version: 1.8
+        java-version: 11
```

Comment:
Do we need Java 11?

Reply:
Unfortunately yes, due to AGP7.0, Java11 is required. https://android-developers.googleblog.com/2020/12/announcing-android-gradle-plugin.html